data/systemd-user/snapd.session-agent.socket

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=REST API socket for snapd user session agent
+
+[Socket]
+ListenStream=%t/snap-session.socket
```

Comment:
If it is for consistency with the systemd unit name, do you want `snapd.session-agent.socket` or `snapd-session-agent.socket`?

Either way, it needs to avoid matching `snap.*` to avoid paths owned by individual snaps.

Reply:
I meant "-",  the parallel is with  snapd-snap.socket in snapd.socket